### PR TITLE
Update reference for validation of Atomic Physics example

### PR DESCRIPTION
**Pull request by ikbuibui**
Update reference data and fix reference generation scripts for the atomic physics example 
Required for JUREAP CI

### CODE REVIEW

**Comment by ikbuibui** [comment]
@BrianMarre Added commit so that we only validate if abundance is above 10^-5. Should we stick to the old reference data?

File: share/picongpu/examples/AtomicPhysics/validation/GenerateReference.py [hunk 1]
```diff
@@ -16,8 +16,8 @@ def generateReference():
         "output_10",
     ]
 
-    results = np.empty((10, 869))
-    for i, output in enumerate(outputs):
+    results = np.empty((10, 26, 869))
```
Comment (PrometheusPi) [inline share/picongpu/examples/AtomicPhysics/validation/GenerateReference.py:19]: I do not see where this increase in dimensions comes from. Was it wrong before?

**Review by PrometheusPi — APPROVED** [review]
Just a questions

File: share/picongpu/examples/AtomicPhysics/validation/GenerateReference.py (hunk 1 above)
Comment (ikbuibui) [reply share/picongpu/examples/AtomicPhysics/validation/GenerateReference.py:19]: Yes, it was a bug in the script before

**Pull request merged**